tools/Config.mk

```diff
@@ -474,7 +474,7 @@ define CLEAN
 endef
 else
 define CLEAN
-	$(Q) rm -f *$(OBJEXT) *$(LIBEXT) *~ .*.swp
+	$(Q) rm -f *$(OBJEXT) *$(LIBEXT) *~ .*.swp $(OBJS) $(BIN)
```

Comment:
but, should we modify the windows case too?

Reply:
Right, I keep forgetting about the other makefile.

Reply:
Ah, you meant the lines above. I just force pushed this.

Reply:
Yes.